### PR TITLE
[15.0][FIX] payroll: Change mail template expressions

### PR DESCRIPTION
**Pull request by victoralmau**
Change mail template expressions according to 15.0

Please @pedrobaeza can you review it?

@Tecnativa

### CODE REVIEW

**Comment by nimarosa** [comment]
@victoralmau the fix is okay we can merge it but please don't bump the version manually, it will be done by the bot when I merge the PR. 

Please change that and we can merge.

**Review by chienandalu — APPROVED** [review]
:+1:

File: payroll/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Payroll",
-    "version": "15.0.1.1.1",
+    "version": "15.0.1.2.0",
```
Comment (nimarosa) [inline payroll/__manifest__.py:5]: Please do not update the version. The bot will do it automatically.

**Review by nimarosa — CHANGES_REQUESTED** [review]

File: payroll/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply payroll/__manifest__.py:5]: The manual bump is correct, as there are migration scripts joined to that version, and launching incorrectly the bot command will make them useless. Please remove your review and merge with `nobump`.

Comment (nimarosa) [reply payroll/__manifest__.py:5]: Yes but I planned to use the "minor" option of the bot to bump to the correct version. 

But okay, we do it like you said.

**Review by nimarosa — APPROVED** [review]

**Pull request merged**